vision.md

```diff
@@ -0,0 +1,52 @@
+Stated simply...
+
+**NUnit 3 is a world-class platform for individuals and teams using a test-based approach to .NET software development.**
+
```

Comment:
This sentence is the actual vision statement. The rest is explanation.

In NUnit 2 it said simply "NUnit is a unit testing framework for developers working with the .NET framework."

IMO this is the most important thing on the page. We need to word it to say what NUnit is going to be.

I would leave it as is or possibly get rid of the notion of a "platform" here and later, since we really don't emphasize it any more.